YesterdayApi/YesterdayApi.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I don't get this change

Reply:
Can you remove this hunk?

Reply:
Any advice on how to do so?

Reply:
In the same way as StyleCop. Rebase from another branch and amend change to previous commit

Reply:
Wouldn't that change the yesterday.csproj file?

Reply:
It will but only comparing to this branch. But comparing to master it wont.